### ISSUE TITLE
Where do I start for BioBERT based RE Tasks on Biomedical data?

### ISSUE THREAD

**Issue by bulutsuzku** [allenai/scispacy#183]
First thank you for enabling this kind of tasks with this great library.

I am new to scispacy, sorry if the question sounds stupid. My goal is to do Relation Extraction on biomedical papers. 

Now on my problem: I want to try scispaCy using a pre-trained model like [BioBERT](https://github.com/naver/biobert-pretrained), then fine-tune with [datasets (these are the train.tsv, dev.tsv, etc) on biomedical RE](https://github.com/dmis-lab/biobert) and finally evaluate.

I already converted the BioBERT weights to pytorch format but I don't know if this is relevant to make it work with the spaCy library?
I guess at some point I should package it for spaCy, but no idea how to go from checkpoint files (which is the format of the BioBERT model) or pytorch formatted files, to spaCy. Is this feasible?

Also the [fine-tuning RE datasets](https://academic.oup.com/view-large/164293067) should be converted right? (I couldn't get the format of the .tsv files, all I could see is the test.tsv files have "index	sentence	label" in the first row. 

Any sample script for a similar task is greatly appreciated.

**Comment by danielkingai2** [comment]
Hi, relation extraction is not something is natively supported with spacy or scispacy. You could try to use NER combined with the dependency parser to extract some relations perhaps. You could check out the spacy pretrain command (https://spacy.io/api/cli#pretrain) perhaps for combining pretrained weights with spacy. Your questions might be better suited for the spacy folks (https://github.com/explosion/spaCy) or the biobert folks (https://github.com/dmis-lab/biobert) depending on your goals.

**Comment by youssef-snoussi** [comment]
Hello @danielkingai2 , I've read recently that there's a library called kindred which is related to spacy and the main goal of it is to extract biomedical relation, any ideas on its efficiency?

**Comment by danielkingai2** [comment]
I don't know anything about this package, sorry.
